tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "outDir": "out/client",
     "rootDir": "src",
     "sourceMap": true,
-    "strict": true
+    "strict": true,
+    "esModuleInterop": true
```

Comment:
Would you clarify the motivation behind adding this?

Reply:
The untildify dependency requires this flag to be set.